src/cell/Cell.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { PropsWithChildren } from 'react';
+import { useTableComponentsContext } from '../TableComponentsContext';
+import { ContentValue } from '../content/ContentValue';
+
+interface CellOwnProps<D, C> {
+  accessor: string | ((row: D) => C);
+}
+
+export type CellProps<D, C> = PropsWithChildren<CellOwnProps<D, C>>;
+export function Cell<D, C>(props: CellProps<D, C>) {
+  const Components = useTableComponentsContext();
+  const { accessor, children } = props;
```

Comment:
More consistency in code, always destruct `props`  at the first line? 
```suggestion
  const { accessor, children } = props;
  const Components = useTableComponentsContext();
```